fastparquet/writer.py

```diff
@@ -671,6 +678,7 @@ def make_metadata(data, has_nulls=True, ignore_columns=[], fixed_text=None,
             se.repetition_type = parquet_thrift.FieldRepetitionType.OPTIONAL
         fmd.schema.append(se)
         root.num_children += 1
+    meta.value = json.dumps(pandas_metadata)
     cats.value = json.dumps(catstruct)
```

Comment:
You are right, this should be removed. On reading I give a warning when this old version is present.